WebApp/LegendPanel.ascx.cs

```diff
@@ -300,48 +300,6 @@ private void AddLayers(Configuration.MapTabRow mapTabRow, AppState appState)
     }
   }
 
-  // removing this fron viwer page 
```

Comment:
What is this code for?

Reply:
this code is shifted in baseMap.cs . this is for basemap and overview layer.

Reply:
now remove this comment